tests/models/test_ModelUtils.py

```diff
@@ -133,6 +133,19 @@ def test_set_at_index(self, seed=0):
             for j in range(index.shape[1]):
                 self._assert_tensor_equal(new_tokens[i, index[i, j]], values[i, j])
 
+    def test_mask_at_index(self, seed=0):
+        torch.manual_seed(seed)
+        index = torch.Tensor([
+            [1, 0, 3, 2],
+            [1, 2, 0, 3],
```

Comment:
Could you make the index have dimension (2, 3) instead of (2, 4)? It should work if index does not have the same shape as tokens.

Reply:
I don't think so, because this mask is used just for equal dimension matrixes as input, right?